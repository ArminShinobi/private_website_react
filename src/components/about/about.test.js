import renderer from 'react-test-renderer';
import About from './about';

it('Verify About Snapshot', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
