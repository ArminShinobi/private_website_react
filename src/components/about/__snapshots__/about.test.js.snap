// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify About Snapshot 1`] = `
<div
  className="background-img"
  style={
    Object {
      "backgroundImage": "url(water_background.jpg)",
    }
  }
>
  <div
    className="d-flex flex-column min-vh-100 justify-content-center align-items-center"
  >
    <div
      className="row m-4"
    >
      <div
        className="col-lg-6 mt-5"
      >
        <img
          alt="Armin Bruckmann"
          className="img-fluid mx-auto d-block img-thumbnail"
          src="profile_picture.jpeg"
        />
      </div>
      <div
        className="col-lg-6 mt-5"
      >
        <h2>
          About Me
        </h2>
        <h4>
          Hi! My name is Armin Bruckmann
        </h4>
        <p
          className="mt-4"
        >
          I graduated from Peter Vischer Gymnasium in 2017 with a grade of 2.5.
        </p>
        <p>
          Currently, I am studying Computer Science at the technical college Georg Simon Ohm, Nuremberg Germany. I expect to finish my Bachelor's degree in the summer semester of 2022.
           
        </p>
        <p>
          I have been working as a student trainee at GfK in Nuremberg, Germany since November 2019. I am developing automated tests for a web application with my team in an agile environment using Robot Framework.
        </p>
      </div>
    </div>
  </div>
</div>
`;
